nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const router = express.Router();
+
+const reservations = require("../data/reservations.json");
+//Respond with the json for all reservations *
+router.get("/", async(request, response) => {
+  try {
+      response.json(reservations);
+      console.log("in/api/reservations");
+  } catch (error) {
+      throw error;
+  }
+});
+//Respond with the json for the reservation with the corresponding id 
+router.get("/:id", async(request, response) => {
+  try {
+      const { id } = Request.params;
```

Comment:
Property 'params' does not exist on type '{ new (input: RequestInfo, init?: RequestInit | undefined): Request; prototype: Request; }'.ts(2339)